modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/release-notes/fragments/release-notes-tabs/index.html

```diff
@@ -6,12 +8,25 @@
 	[#assign currentTabId = (currentDisplayPaths?first)! /]
 
 	[#assign currentDisplayIds = (displayPortalURLs?last?split(currentTabId)?last)! /]
+
+	[#assign
+		parameterName = "start"
```

Comment:
@samuelleolopes since we are only using parameter name and value variables only once i think its cleaner if you just write the values where they are used